docs/en/your-profile-and-preferences/log-out-of-status.md

```diff
@@ -0,0 +1,29 @@
+---
+id: 387
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Log out of Status
+
+When you decide not to use Status on the current device, you can log out to keep your profile secure. Your profile automatically logs out if you uninstall Status or sometimes after you update Status.
+
+!!! Caution
+    If you log out without [backing up the recovery phrase][back-up-and-secure-your-recovery-phrase] and saving your password, you can't access your Status profile or the associated data and funds again. Keep the password and the recovery phrase in a safe place. Status cannot recover or reset them for you.
+
+=== "Mobile"
+
+    1. Tap your avatar.
+    1. Scroll down to the bottom of the Settings list and tap :mobile-log-out: **Log out**.
```

Comment:
Is the word "Settings" visible in the screen? Then, it should go in bold typeface:

> ...the bottom of the **Settings** list...

See https://write.status.im/style-conventions/#format-guidelines for reference.